src/db/sqlite/lrsql/sqlite/sql/ddl.sql

```diff
@@ -438,7 +438,8 @@ CREATE TABLE IF NOT EXISTS reaction (
   ruleset    BLOB NOT NULL,             -- serialized reaction spec
   created    TIMESTAMP NOT NULL,        -- timestamp
   modified   TIMESTAMP NOT NULL,        -- timestamp
-  active     INTEGER                    -- true/false/null - active/inactive/soft delete
+  active     INTEGER,                   -- true/false/null - active/inactive/soft delete
```

Comment:
not sure how i feel about the trinary vs another field for deleted, I'm usually not a fan, but I guess I can't practically see an issue with it.

Reply:
Yep, that is why we are soft-deleting. As far as the trinary goes, we used it on previous projects so I thought it was OK, but am happy to change if desired.